components/recurring-contributions/RecurringContributionsContainer.js

```diff
@@ -42,25 +39,27 @@ const RecurringContributionsContainer = ({ recurringContributions, filter, creat
   }
 
   return (
-    <AllCardsContainer my={2}>
-      {displayedRecurringContributions.map(contribution => (
-        <CollectiveCardContainer key={`${contribution.id}-container`}>
-          <RecurringContributionsCard
-            key={contribution.id}
-            mx={3}
-            width={250}
-            height={360}
-            collective={contribution.toAccount}
-            status={contribution.status}
-            contribution={contribution}
-            style={{ position: 'relative' }}
-            createNotification={createNotification}
-            account={account}
-            data-cy="recurring-contribution-card"
-          />
-        </CollectiveCardContainer>
-      ))}
-    </AllCardsContainer>
+    <Container maxWidth={Dimensions.MAX_SECTION_WIDTH} pl={Dimensions.PADDING_X} mt={4} mx="auto">
+      <Flex flexWrap="wrap" justifyContent={['space-evenly', 'left']} my={2}>
+        {displayedRecurringContributions.map(contribution => (
+          <CollectiveCardContainer key={`${contribution.id}-container`}>
```

Comment:
**[:ice_cream: Details & personal preferences]** Adding this suffix would be great for `data-cy`, but `key` only cares about uniqueness (it's not used elsewhere).

```suggestion
          <CollectiveCardContainer key={contribution.id}>
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>